vendor/idrive_vnc.js

```diff
@@ -36,4 +38,75 @@ function handleConnect() {
   if ( myParam !== null && myParam.length > 0) {
     document.getElementById('status').textContent = "Connected to - " + myParam;
   }
+
+  // Handle Clipboard Paste
+  let clipboardButton = document.getElementById('sendClipboardButton');
```

Comment:
I don't understand the code here, so can't comment on whether it's working or not. I hope its not a copy paste from someone else's code.

Reply:
@diganta-dutta the idea is borrowed from `Vultr` and the code is on the same pattern. We can discuss and I can make changes accordingly. I will try to make some changes so it doesn't look same.

Reply:
@diganta-dutta I have made the code changes and simplified the code. Reading text from clipboard is standard function ( [refer to example here](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/read)) and then we need to user sendkey function based upon the character.